### DISCUSSION TITLE
Sites.Selected does not give full control

### DISCUSSION THREAD

**Discussion by wonderplayer** [pnp/pnpcore#944] [Q&A] [answered]
Hello,

Most likely not for this repository, but I'm trying to update item system properties.
Question about Sites.Selected permissions.
Using PnPPowerShell I could achieve update, but only with regular update, system update didn't work.

So, with PnP.Core I'm also trying to update list item Author, Editor, Created and Modified dates, and can't succeed :(
With Sites.FullControl.All I can do it.

I've given write permissions to the app in the site. I see that there is owner permission in the schema as well, but Graph shows me that this role is not correct.

Maybe you have some suggestions how I can achieve this with Sites.Selected permission?

Thank you!

**Comment by jansenbe** [comment, marked as answer]
I think you need `Sites.FullControl.All` for updating those properties, see https://github.com/pnp/pnpcore/discussions/672
